package.json

```diff
@@ -37,7 +37,7 @@
     "stylelint-project": " stylelint -- 'src/**/*.{ts,tsx}'",
     "prettier-project": "yarn run prettier-write 'src/**/*.{ts,tsx}'",
     "prettier-write": "yarn run prettier --write",
-    "relay": "relay-compiler --src ./src --schema data/schema.graphql --language typescript --artifactDirectory ./src/__generated__",
+    "relay": "relay-compiler --src ./src --schema data/schema.graphql --language typescript --artifactDirectory ./src/__generated__ --exclude '**/node_modules/****/__mocks__/**,**/__generated__/**'",
```

Comment:
What does `****` and `**, **` do?

Reply:
Hmmmm, this is copy-🍝 from the `relay-compiler` source which was excluding tests, but I think I missed inserting a comma between `****`, these should be 3 exclude patterns.

Great catch! 👌